must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -56,3 +56,15 @@ do
         { timeout 120 oc -n "${ns}" logs --all-containers "${pod}" &> "${LOG_DIR}"/"${pod}".log; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
     done
 done
+
+# Add important notfications to a notifications.txt file at the root of the noobaa collection path
+UNMANAGED_NOOBAA_NOTIF="The noobaa deployed on this cluster is not managed by the storagecluster CR and will not react to configuration changes made to the CR"
+
+NOOBAA_RECONCILE_STRATEGY=$( oc -n ${ns} get storagecluster -o jsonpath='{.items[0].spec.multiCloudGateway.reconcileStrategy}' 2> /dev/null)
+if [ "${NOOBAA_RECONCILE_STRATEGY}" ] && [ "${NOOBAA_RECONCILE_STRATEGY}" != "managed" ]
```

Comment:
Do we need both of these tests? Wouldn't the second one already cover the first since the first will only ever fail when it's `""`, which is `!= "managed"`?

Reply:
Actually, wait, this is even easier. You need to check for `standalone` or `ignore`, not `manage`. Anything that is not one of those two will be treated the same as `manage` (the implicit default).

Reply:
The condition here is `if a value exists and the value is not manage` 
If we remove the first section we will get `if the value is not manage` 
which differ under the case where the value does not exist

Reply:
Yes, you don't want that. All you want is "is the value `ignore` or `standalone`?". Everything that is not those two words is equivalent to "manage".

Reply:
done